python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -44,8 +44,17 @@ def _build_fil_classifier(m, data, arg={}):
         raise ImportError("No XGBoost package found")
 
     # use maximum 1e5 rows to train the model
-    train_size = min(data[0].shape[0], 100000)
-    dtrain = xgb.DMatrix(data[0][:train_size, :], label=data[1][:train_size])
+    train_size = min(data[0].shape[0], 1000)
+
+    if isinstance(data[0], np.ndarray):
+        dtrain = xgb.DMatrix(data[0][:train_size, :], label=data[1][:train_size])
+    else:
+        # if the input is gpuarray
+        print(type(data[0]))
+        train_data = np.ascontiguousarray(data[0][:train_size, :].copy_to_host())
```

Comment:
This won't work for a pandas dataframe, which can also be an input.

Reply:
Yep, pandas should be supported now. However, I'm getting `TypeError: X matrix format <class 'pandas.core.frame.DataFrame'> not supported` error in `predict()`. Does FIL support pandas df as input? or maybe I did something wrong